packages/itwinui-css/src/icon/mixins.scss

```diff
@@ -3,50 +3,45 @@
 @import '../style/index';
 
 @mixin iui-icons-default {
```

Comment:
Maybe we should consider renaming it? Because now when there is color and display mixins, default doesn't tell anything, I had to look at the implementation what it does 😀 Maybe rename it to `-medium` or smth, but that is just my opinion.

Reply:
Or we can add `size` suffix to make it clear.
But `medium` could work too.

Reply:
I also was thinking we could have `size` word in the name.

Reply:
Actually, here's an idea I like better:
```scss
/// @arg $size Can be one of: s, m, l, xl, 2xl, 3xl
@mixin iui-icon-size($size: m) {
	@if $size == 's' {
	  width: $iui-icons-small;
	  height: $iui-icons-small;
	}
	@if $size == 'm' {
	  width: $iui-icons-default;
	  height: $iui-icons-default;
	}
	@if $size == 'l' {
	  width: $iui-icons-large;
	  height: $iui-icons-large;
	}
	@if $size == 'xl' {
	  width: $iui-icons-xl;
	  height: $iui-icons-xl;
	}
	@if $size == '2xl' {
	  width: $iui-icons-xxl;
	  height: $iui-icons-xxl;
	}
	@if $size == '3xl' {
	  width: $iui-icons-3xl;
	  height: $iui-icons-3xl;
	}
}
```

Reply:
Implemented. Mostly